app/components/support_interface/application_choice_component.rb

```diff
@@ -181,7 +181,7 @@ def visible_over_register_api?
     end
 
     def change_course_choice_link
-      return {} unless @application_choice.application_form.editable? && ApplicationStateChange::DECISION_PENDING_STATUSES.include?(application_choice.status.to_sym)
+      return {} unless !FeatureFlag.active?(:hide_support_change_course_tool) && @application_choice.application_form.editable? && ApplicationStateChange::DECISION_PENDING_STATUSES.include?(application_choice.status.to_sym)
```

Comment:
you can now enjoy the glory of `FeatureFlag.inactive?`

Reply:
🤩